byterun/Makefile.shared

```diff
@@ -37,6 +37,26 @@ endif
 
 CC=$(BYTECC)
 
+ifdef BOOTSTRAPPING_FLEXLINK
+CFLAGS=-DBOOTSTRAPPING_FLEXLINK
+else
+CFLAGS=
+endif
+
+# On Windows, OCAML_STDLIB_DIR needs to be defined dynamically
+
+ifeq "$(UNIX_OR_WIN32)" "win32"
+CFLAGS += -DOCAML_STDLIB_DIR='"$(LIBDIR)"'
+endif
+
+CFLAGS += $(IFLEXDIR) $(BYTECCCOMPOPTS)
+
+DFLAGS=$(CFLAGS) -DDEBUG
+
+ifneq "$(CCOMPTYPE)" "msvc"
```

Comment:
As far as I can tell, this changes the behaviour of the non-msvc builds and fixes an issue where non-msvc debug builds didn't have "-g" in $(DFLAGS). Am I right?